### PR TITLE
Replace native date input by react-datepicker

### PR DESCRIPTION
**Pull request by notAro14**

### CODE REVIEW

File: src/fields/DateField.jsx [hunk 1]
```diff
@@ -1,12 +1,61 @@
...
+        ref={ref => (this.datepicker = ref)}
+        selected={value ? new Date(value) : null}
+        onChange={date => {
+          this.setState({ date })
+          onChange(date ? format(date, 'yyyy-MM-dd') : '')
+        }}
+        isClearable
+        locale={locale}
+        dateFormat={dateFormat}
+        className="Formol_InputField Formol_DateField Formol_Field__element"
```
Comment (cmigazzi) [inline src/fields/DateField.jsx:51]: `formol` uses `bemboo` for class names management, it should be good to use it here too (via the `@block` decorator).

File: test/formol/types/datepicker.test.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        .first()
+    const submit = () => wrapper.find('.Formol_Formol__submit')
+    const cancel = () => wrapper.find('.Formol_Formol__cancel')
+
+    expect(submit().props().disabled).toBeTruthy()
+    expect(cancel().props().disabled).toBeTruthy()
+    expect(input().props().type).toEqual(nativeType)
+    expect(input().props().value).toEqual(value1)
+
+    await input().simulate('focus')
```
Comment (cmigazzi) [inline test/formol/types/datepicker.test.jsx:37]: This line causes an `UnhandledPromiseRejectionError`. Without it, all tests are passed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+### [2.9.0](https://github.com/Kozea/formol/compare/v2.8.3...v2.9.0)
+
+- Replace native date input by react-datepicker
+
```
Comment (cmigazzi) [inline CHANGELOG.md:4]: I think this should be in master branch after merging the PR, as well as the version publishing commit.

**Review by cmigazzi — CHANGES_REQUESTED** [review]
Wouldn't it be better to keep the native date field and just add the new datepicker as a `DatpickerField` ? It lets people choose which one to use.

**Comment by notAro14** [comment]
> Wouldn't it be better to keep the native date field and just add the new datepicker as a `DatpickerField` ? It lets people choose which one to use.

Yes, actually it was the initial plan
1. set the date field as datepicker component
2. rename the native input as date-native or alike

I have to work on 2 yet. I almost forgot 😅

File: src/fields/DateField.jsx (hunk 1 above)
Comment (notAro14) [reply src/fields/DateField.jsx]: Fait dans f88852f

**Review by cmigazzi — APPROVED** [review]
:+1:

**Review by gentooboontoo — APPROVED** [review]
:100:

**Pull request merged**